pubsub/noxfile.py

```diff
@@ -23,12 +23,6 @@
 
 LOCAL_DEPS = (os.path.join("..", "api_core"), os.path.join("..", "core"))
 
-BLACK_PATHS = ["docs", "google", "tests", "noxfile.py", "setup.py"]
```

Comment:
Hey @anguillanneuf I see recent changes to our templates being undone here. Would you mind installing synthtool from head and re-running? 

```python3 -m pip install --user --upgrade git+https://github.com/googleapis/synthtool.git```

Thank you!

Reply:
I just used the latest synthtool. How does it look now?